### ISSUE TITLE
Make surface nets the default closed surface generation algorithm

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#8093]
As discussed, we should use surface nets to generate closed surface representation from binary labelmap represenation in segmentations. See the discussion at https://discourse.slicer.org/t/bad-performance-when-segmenting-11gb-micro-ct/38362/8?u=lassoan

**Comment by muratmaga** [comment]
However, there is this issue to keep debug: 
https://discourse.slicer.org/t/surface-normal-calculation-issue-using-surfacenets/40811

**Comment by jamesobutler** [comment]
@lassoan Do we want to backport any of Spiros work from https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11847 to then use surface nets by default?

**Comment by jcfr** [comment]
Since this change also depends on [MR-12452](https://gitlab.kitware.com/vtk/vtk/-/merge_requests/12452) (that has been integrated), the backport is not specific to only `Filters/Core/vtkSurfaceNets3D.*` files and backporting all the relevant changes may have unintended side effects.

For this reason, I suggest we postpone this for the next release cycle.

**Comment by jcfr** [comment]
cc: @spyridon97  @lassoan

**Comment by lassoan** [comment]
OK, it makes sense. It is better to test it for a while in preview releases anyway.

**Comment by spyridon97** [comment]
The face normal fix does not depend on https://gitlab.kitware.com/vtk/vtk/-/merge_requests/12452. If needed, i could create a patch for a specific version of VTK (that you want) that only fixes the face normals.

**Comment by jcfr** [comment]
> i could create a patch for a specific version of VTK

That would work well 🙏 . Ideally, the patch should apply to this branch: https://github.com/Slicer/VTK/tree/slicer-v9.5.2-2025-09-16-7c0494a68

**Comment by spyridon97** [comment]
I will take care of it on monday.

**Comment by jcfr** [comment]
@lassoan If you have a chance to test between now and tomorrow, I have no issue integrating this (and even making it the default)

**Comment by jcfr** [comment]
@spyridon97  Thanks for the guidance 🙏 , I do think I was able to integrate the specific patch you referenced when we talked offline.

Note that, I kept the update of `offsetIter` as I didn't backport kitware/VTK@e17ec0cb52 ("Filters/IO/Tests: Use FixedSize offsets to save space", 2025-09-11)

**Comment by lassoan** [comment]
OK, I'll test this.

**Comment by jcfr** [comment]
🙏

**Comment by lassoan** [comment]
I have changed the default in my local branch and testing the behavior now. So far everything looks good.

**Comment by jcfr** [comment]
@lassoan  Once you approve https://github.com/Slicer/Slicer/pull/8831, it can be merged. Then you could follow-up with PR changing the default and that would close this issue.

**Comment by lassoan** [comment]
The generated surfaces look less smooth. It may be due to the surface geometry is actually less smooth and/or the surface normals are not computed.

The surface net internal smoothing produces somewhat different results compared to windowed since. This needs a bit more work.

After exporting to model, some scalar array remains in the polydata, which throws off the coloring of the model by default (coloring by scalar gwys enabled).

All these would worth some more investigation before we change the default method to surface nets. Therefore it might make sense to change the default in Slicer 5.11.

**Comment by jcfr** [comment]
> All these would worth some more investigation before we change the default method to surface nets. Therefore it might make sense to change the default in Slicer 5.11.

Agreed. Thanks for taking the time to investigate and look into this 🙏
